its/plugin/src/integrationTest/java/org/sonarsource/iac/MeasuresTest.java

```diff
@@ -20,40 +20,68 @@
 package org.sonarsource.iac;
 
 import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+@Execution(ExecutionMode.CONCURRENT)
 class MeasuresTest extends TestBase {
 
-  private static final String BASE_DIRECTORY = "projects/measures/";
+  protected static final String BASE_DIRECTORY = "projects/measures/";
+
+  @Nested
+  class CloudFormationMeasuresTest {
+    @ParameterizedTest
+    @CsvSource({
+      "cloudformationYamlMeasures, cloudformation, file1.yaml, 2, 5, 1, 1=1;4=1;5=1;6=1;7=1",
+      "cloudformationJsonMeasures, cloudformation, file1.json, 2, 9, 0, 1=1;2=1;3=1;4=1;5=1;6=1;7=1;8=1;9=1",
+    })
+    @Execution(ExecutionMode.SAME_THREAD)
```

Comment:
Because some tests access the same folder, they have to be run sequentially. The structure `main class in parallel -- nested classes enclosing parameterized tests sequential` achieves that